lib/rawQueries.js

```diff
@@ -24,11 +26,18 @@ const BuildFactoryQueries = {
         WHERE rank = 1
         ORDER BY "jobId", "id" DESC`,
 
-    latestBuildQueryMySql: `SELECT * FROM (
-        SELECT *, RANK() OVER ( PARTITION BY jobId ORDER BY id DESC ) AS \`rank\`
-        FROM builds WHERE eventId in
-        (SELECT id FROM events WHERE groupEventId = (:groupEventId))) AS \`events\`
-        WHERE \`rank\` = 1
+    latestBuildQueryMySql: `SELECT id, environment, eventId, jobId, parentBuildId, number, container, cause, sha,
```

Comment:
this query is fired when a build restarted. @wahapo please help check this as well.

Reply:
No error occurred and the build started. This query fired when the Aggregate View was displayed and when the build was (re)started from page of Aggregate View.
```
[
  [
    TextRow {
      id: 1,
      jobId: 1,
      status: 'ABORTED',
      startTime: null,
      endTime: null
    },
    TextRow {
      id: 2,
      jobId: 1,
      status: 'QUEUED',
      startTime: null,
      endTime: null
    },
    TextRow {
      id: 3,
      jobId: 1,
      status: 'QUEUED',
      startTime: null,
      endTime: null
    },
    TextRow {
      id: 4,
      jobId: 1,
      status: 'QUEUED',
      startTime: null,
      endTime: null
    },
    TextRow {
      id: 5,
      jobId: 1,
      status: 'ABORTED',
      startTime: null,
      endTime: null
    }
  ],
  [
    TextRow {
      id: 1,
      jobId: 1,
      status: 'ABORTED',
      startTime: null,
      endTime: null
    },
    TextRow {
      id: 2,
      jobId: 1,
      status: 'QUEUED',
      startTime: null,
      endTime: null
    },
    TextRow {
      id: 3,
      jobId: 1,
      status: 'QUEUED',
      startTime: null,
      endTime: null
    },
    TextRow {
      id: 4,
      jobId: 1,
      status: 'QUEUED',
      startTime: null,
      endTime: null
    },
    TextRow {
      id: 5,
      jobId: 1,
      status: 'ABORTED',
      startTime: null,
      endTime: null
    }
  ]
]
```

Reply:
thnx, then good to merge once approved.